_includes/nav.html

```diff
@@ -16,8 +16,7 @@
         <li {% if page.url contains "/libraries" %}class="active"{% endif %}><a href="{{"/libraries/" | prepend: site.baseurl }}">LIBRARIES</a></li>
         <li {% if page.url contains "/community" %}class="active"{% endif %}><a href="{{"/community/" | prepend: site.baseurl }}">COMMUNITY</a></li>
         <li {% if page.url contains "/news" %}class="active"{% endif %}><a href="{{"/news/" | prepend: site.baseurl }}">NEWS</a></li>
-        <li><a href="http://github.com/scala-js/scala-js"><i aria-label="github" style="font-size: 20px" class="fa fa-github"></i></a></li>
-        <li><a href="http://twitter.com/scala_js"><i aria-label="twitter" style="font-size: 20px" class="fa fa-twitter"></i></a></li>
+        <li><a href="#" id="search-icon"><i style="font-size: 20px" class="fa fa-search"></i></a></li>
```

Comment:
Hum ... if I add them back, I don't have any problem, here. And still plenty of space to fill.